### PR TITLE
Session model

### PR DESCRIPTION
**Pull request by METrimble**
## Description
Session model Session.js in Sequelize with an id, user id, and expiring time. The user id attribute will be uncommented and act as a foreign key to users when we connect the models together. CheckIfExpired is a function included as well which checks if the current date is after the expiring date. 

The session tests are located in session.jest.js. This includes two tests; one which tests if we can make a session row with the default expiration time, and another which checks if the CheckIfExpired function functions correctly. If needed, I can add another to check if the checkIfExpired function works in dates that should be expired. 

### Special Notes
N/A

### How has this been tested?
using the session.jest.js file described above. To note, I wasn't able to get the "npm test" command to work on my own branch and master. I ran the command: $ node_modules/.bin/jest --runInBand and I was able to get the tests to run. For whatever reason NODE_ENV is not recognized so I was unable to use the npm test script, and I had to make a path to the jest library. 

## Requirements
Fill out the checkboxes below, verifying that all the requirements have been satisfied:
- [X] Have I written new tests to show my code works?
- [X] Do all other tests still work?
- [X] Have I documented where necessary (technical docs, how to docs, etc.)?
- [X] Have I requested at least 1 reviewer?
- [X] Have I left comments so my work is easy to understand?
- [X] Have I looked over my code again to weed out any mistakes I may have made or details missed?
- [X] Does my work reflect all the needs of this given task?

### CODE REVIEW

**Review by mitchellstewart7 — COMMENTED** [review]
I think it would be a good idea to have a test where a session actually is expired. I'm thinking something along the lines of 
1. create session at a moment 3:59:50 before the time the session is actually created
2. delay(15 seconds)
3. check if the session is expired
And have that in the jest tests.

Other than that I think it looks good and everything runs correctly :)

File: __tests__/app/models/session.jest.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const db = require('../../../app/models/index')
+const moment = require('moment')
+
+describe("Session model", () => {
+    let Session
+
+    beforeAll(async() => {
+        await db.sequelize.sync() // connect to the database
```
Comment (ghost) [inline __tests__/app/models/session.jest.js]: This shouldn't be needed for the tests anymore - I made an initialization script that automatically connects to the database before any tests are run :)

File: __tests__/app/models/session.jest.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const db = require('../../../app/models/index')
+const moment = require('moment')
+
+describe("Session model", () => {
+    let Session
```
Comment (ghost) [inline __tests__/app/models/session.jest.js]: I think you meant to have lowercase "session"

File: __tests__/app/models/session.jest.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            expect(moment(session.expires).Empty).toBeFalsy()
+        })
+
+        it ("Should return false, as we just made the session", async() => {
+            expect(session.checkIfExpired()).toBeFalsy()
+        })
+    })
+    
+
+    afterAll(async () => {
```
Comment (ghost) [inline __tests__/app/models/session.jest.js:36]: Same with this - moved it to an after-the-fact closing of connection. Should probably just call "session.destroy()" so the session you created in the beforeAll is deleted

**Review by ghost — CHANGES_REQUESTED** [review]
I left a few little comments on the test file. I also agree with Mitch's comments. This is great work, thank you Elayne!

File: db/migrations/20230202034152-create-sessions.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+
+    await queryInterface.createTable('Users', {
```
Comment (ghost) [inline db/migrations/20230202034152-create-sessions.js]: I think you meant 'Sessions'

File: db/migrations/20230202034152-create-sessions.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  },
+
+  async down (queryInterface, Sequelize) {
+    /**
+     * Add reverting commands here.
+     *
+     * Example:
+     * await queryInterface.dropTable('users');
+     */
+    return queryInterface.dropTable('Session');
```
Comment (ghost) [inline db/migrations/20230202034152-create-sessions.js]: plural: 'Sessions'

**Review by ghost — APPROVED** [review]

**Pull request merged**